tests/setup.js

```diff
@@ -64,7 +64,7 @@ beforeEach(() => {
 
 afterEach(() => {
     if (logs.length > 0 || warns.length > 0 || errors.length > 0) {
-        throw new Error('Unexpected console logs' + logs + warns + errors);
+        // throw new Error('Unexpected console logs' + logs + warns + errors);
```

Comment:
Why are you disabling it?

Reply:
Oops, sorry, nice catch. That's for my own debugging only.  Will revert that.

Reply:
Thanks @jespino, now updated.